incubator/mysqlha/values.yaml

```diff
@@ -82,3 +82,11 @@ metrics:
     requests:
       cpu: 100m
       memory: 100Mi
+
+
+service:
+  ## Specify the nodePort value for the LoadBalancer and NodePort service types.
```

Comment:
nit: Is this comment necessary as it's just the definition of a node port?

Reply:
Oh, It's my fault, I will be careful more.